ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -86,16 +86,26 @@ then
 			echo "ERROR [init] format failed"
 		fi
 	fi
+else
+#
+# Is there a device mounted for storage
+#
+	if mft query -q -k MTD -s MTD_PARAMETERS -v /mnt/microsd
+	then
+		set PARAM_FILE /fs/microsd/params
```

Comment:
Use the $PARAM_FILE instead.

Reply:
```suggestion
		set $PARAM_FILE /fs/microsd/params
```
Like this?

Reply:
Actually these two lines can be removed

Reply:
I mean that https://github.com/PX4/PX4-Autopilot/blob/ed5a7e63c8df6863cd9fc1e97c886d9ccb12701b/ROMFS/px4fmu_common/init.d/rcS#L134
Already sets PARAM_FILE to /fs/microsd/params so that code is redundant.